apps/dashboard/src/app/api/testnet-faucet/claim/route.ts

```diff
@@ -17,6 +17,22 @@ export const POST = async (req: Request) => {
     throw new Error("Invalid chain ID.");
   }
 
+  const {
+    THIRDWEB_ENGINE_URL,
+    NEXT_PUBLIC_THIRDWEB_ENGINE_FAUCET_WALLET,
+    THIRDWEB_ACCESS_TOKEN,
+  } = process.env;
```

Comment:
Can't destructure process.env object 
https://nextjs.org/docs/app/api-reference/next-config-js/env

Reply:
The way we are using it now is building correctly, but I can move the vars. Should I create a new env config in the `next.config.js` file? here: https://github.com/thirdweb-dev/js/blob/main/apps/dashboard/next.config.js

Reply:
probably just having it as 3 const is enough?